dotCMS/src/main/java/com/dotcms/publisher/pusher/PushPublisher.java

```diff
@@ -5,6 +5,7 @@
 import java.io.FileInputStream;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.FileInputStream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)